exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -368,6 +368,13 @@ drill.exec: {
     // planning and managing queries. Primarily for testing.
     cpus_per_node: 0,
   }
+  # Grace period is the amount of time where the drillbit accepts work after
+  # the shutdown request is triggered. The primary use of grace period is to
+  # avoid the race conditions caused by zookeeper delay in updating the state
+  # information of the drillbit that is shutting down. So, it is advisable
+  # to have a grace period that is atleast twice the amount of zookeeper
+  # refresh time.
+  grace_period : 10000
```

Comment:
Units? Just add a note to the end of your comment to say the units are ms. In the future, I've found it to be handy to put the units in the name: `grace_period_ms` for example.